aten/src/ATen/native/native_functions.yaml

```diff
@@ -1858,7 +1858,16 @@
 - func: cudnn_batch_norm(Tensor input, Tensor weight, Tensor? bias, Tensor? running_mean, Tensor? running_var, bool training, float exponential_average_factor, float epsilon) -> (Tensor, Tensor, Tensor, Tensor)
   dispatch:
     CUDA: cudnn_batch_norm
-  autogen: cudnn_batch_norm.out
+
+# To pass the check in `add_generated_native_functions`,
+# param `reserve` is set as a placeholder, just passing an empty tensor for it.
+# usage:
+# auto reserve = torch::empty({0}, torch::device(torch::kCUDA));
+# at::native::cudnn_batch_norm_out(..., epsilon, output, save_mean, save_var, reserve);
+# TODO: If possible, rename the out0 ... out3 to output, save_mean, save_var, reserve (bc break)
```

Comment:
Feel free to remove this comment as I'm not sure people will see it.

Reply:
Done~